test-data/unit/check-based-default-return.test

```diff
@@ -0,0 +1,37 @@
+-- Type checker test cases for default return type.
+
+[case testDefault]
+# based
```

Comment:
whats with the `# based` comments i thought it just checks for "based" in the test name

Reply:
Thanks, this was committed by accident 😳